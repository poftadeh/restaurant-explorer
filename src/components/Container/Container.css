.app-container {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: #fff;
  border-radius: 4px;
  padding: 4rem;
  box-shadow: 0 1.5rem 4rem rgba(0, 0, 0, 0.15);
  width: 100rem;
  height: 70rem;
}
