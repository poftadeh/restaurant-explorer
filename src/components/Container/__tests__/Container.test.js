import React from 'react';
import { shallow } from 'enzyme';
import Container from '../index';

it('renders <Container /> properly', () => {
  const wrapper = shallow(<Container />);
  expect(wrapper).toMatchSnapshot();
});
