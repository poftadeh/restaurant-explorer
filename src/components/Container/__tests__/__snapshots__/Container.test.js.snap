// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Container /> properly 1`] = `
<div
  className="app-container"
/>
`;
